tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -123,15 +123,15 @@ Object.PCM.pcm [
 # top-level pipeline connections
 Object.Base.route [
 	{
-		sink 'dai-copier.HDA.2.1'
+		sink 'dai-copier.HDA.$HDA_ANALOG_DAI_NAME.playback'
 		source 'gain.2.1'
 	}
 	{
 		source 'mixin.1.1'
 		sink 'mixout.2.1'
 	}
 	{
-		source 'dai-copier.HDA.4.1'
+		source 'dai-copier.HDA.$HDA_ANALOG_DAI_NAME.capture'
```

Comment:
if we need to reduce the widget name length, "playback" and "capture" are obvious targets.

We could use something like "p"/"c" (as done for ALSA in /proc pcm0c/   pcm0p/   pcm10p/  pcm1p/). Or 'pb' and 'cp'.

Just a suggestion.